package.json

```diff
@@ -6,7 +6,7 @@
   "license": "MIT",
   "main": "./dist/electron/main.js",
   "scripts": {
-    "build": "node .electron-vue/build.js && electron-builder -wml",
+    "build": "node .electron-vue/build.js && electron-builder -l",
```

Comment:
we have the `build-linux` script below which uses -l flag.

Reply:
Thank you for pointing that out, sorry tunnel vision :D

Reply:
no worries, thanks!